City-of-Helsinki/drupal-hdbt
# UHF-2499: Fix missing padding from containers like banners.

Removed a piece of code that was introduced in https://github.com/City-of-Helsinki/drupal-hdbt/pull/139 and caused missing left padding on containers that should have it. (ie. map and banner)

The original code tried to fix a problem of extra padding on non-banner text content. Sadly how the code is built, that solution needs to be redone more carefully. Probably a major rework of the container styles is needed. It was agreed that the time for this rework is not now, but we should remove this padding-breaking change for now and fix the bigger underlying issues at a later date.

How to test (existing install)

1. Check a local page that has navigation and banners. Currently the left padding of banners/maps, etc is missing.
2. On project root run `composer require drupal/hdbt:dev-UHF-2499_container_padding`
3. `make drush-cr`
4. Check the page again, the missing padding should be now back.
